build_tools/nb_json_formatter.py

```diff
@@ -0,0 +1,51 @@
+"""Notebook formatter.
+
+Formats jupyter notebooks as follows:
+
+* changes execution count to consecutive numbers, starting with 1
+* sets json indent to 1 space
```

Comment:
I'm a bit surprised by 1. Usually 2, 4, etc. are used, although I can't think of anything against 1.

Reply:
it is simply the status quo, if you open the `ipynb` notebooks with a `json` editor.